adapter/build.gradle.kts

```diff
@@ -9,10 +9,15 @@
 
 plugins {
     id("spha-kotlin-conventions")
+    alias(libs.plugins.serialization)
```

Comment:
2/3 modules are now using serialization. 
we could create a new "spha-serialization-conventions" plugin and use this instead of manually including it in both modules. But this is really minor and up to you. When we end up using it in all 3 modules we should definitely move it into the Kotlin-conventions plugin.